DataFormats/L1TCorrelator/interface/TkElectronFwd.h

```diff
@@ -10,6 +10,7 @@
 #include <vector>
 #include "DataFormats/Common/interface/Ref.h"
 #include "DataFormats/Common/interface/RefVector.h"
+#include "DataFormats/L1TParticleFlow/interface/RegionalOutput.h"
```

Comment:
it looks like this creates a circular dependency between DataFormats/L1TParticleFlow and DataFormats/L1TCorrelator packages

Reply:
I have moved it to DataFormats/L1Trigger